lib/utils/math.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+function getMinMax(nums) {
+  let min = nums.length ? Math.min(...nums) : 0;
+  let max = nums.length ? Math.max(...nums) : 0;
+  return { min, max };
+}
+
+function getMean(nums) {
+  return nums.reduce((total, num) => total + num, 0) / nums.length;
+}
+
+function getPercentileMean(nums, per) {
+  let index = Math.floor((per / 100) * nums.length);
+  return getMean(nums.sort().slice(0, index));
```

Comment:
sometimes if there is only one build then this try to get `slice(0,0)`, ultimately it will become NaN.

we can use `Math.max(index, 1)` ?